### DISCUSSION TITLE
Kolaborasi Komunitas

### DISCUSSION THREAD

**Discussion by dinioktaviani** [kawalcovid19/wargabantuwarga.com#620] [Q&A] [answered]
Halo! Bagaimana ya caranya jika komunitas saya ingin menjadi kolaborator bagi #wargabantuwarga? Dan kemana saya bisa mengontak lebih lanjut?

**Comment by moemura** [comment]
Di situsnya sendiri ada tombol **Hubungi Kami** di bawah, silakan dicoba.

**Reply by zainfathoni** [reply]
Tombol yang itu bukan untuk kolaborasi, tapi untuk menghubungi Hotline kami terkait dengan kasus COVID-19 yg diderita. 😅🙏

**Comment by MFaizghifari** [comment, marked as answer]
Halo @dinioktaviani, perkenalkan saya Faiz Project Manager untuk overall inisiatif WBW. Untuk kolaborasi boleh menghubungi saya via email ini ya: faizghifariproject@gmail.com
